packages/painperdu/src/PainPerdu/painPerdu.tsx

```diff
@@ -1,20 +1,40 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useState } from 'react'
+import { PainPerduModal }  from '../PainPerduModal/PainPerduModal'
+import '../index.css'
 
 export const PainPerdu = () => {
-  const [isModal, setModal] = useState<boolean>(false);
+  const [isModal, setModal] = useState<boolean>(false)
+	const [isContainerBc, setIsContainerBc] = useState<boolean>(false)
 
 	const handleEsc = (event: KeyboardEvent): void => {
 		const isMetaKey: boolean = event.metaKey
-		if (event.code === 'KeyK' && isMetaKey) setModal(true)
+		if (isMetaKey && event.code === 'KeyK') {
```

Comment:
Plutôt utiliser la négation içi c'est plus performant je m'explique la on verifie que quand une key est push la key soit de type `keyK` ou `Escape` et le moteur js va passer dans les deux if pour vérifier si on utilise la négation on stoppe l'exécution de la function avant les deux if ce qui est plus performant ça donne ça:

```suggestion
const keyPressed = event.code;
const isMetaKey = event.metaKey;
const commands = {
  openModal: 'KeyK',
  closeModal: 'Escape',
};

if (
  (keyPressed !== commands.openModal && isMetaKey) || 
  (keyPresses !== commands.closeModal && isMetaKey)
) return;

if (keyPressed === commands.openModal) {
  setModal(true)
  setIsContainerBc(true)
  return;
}

setModal(false)
setIsContainerBc(false)
```

T'en penses quoi ?

Reply:
Corrected : [d470d1f](https://github.com/luctst/painperdu/pull/9/commits/d470d1fae23239f3cb3ede4c9821024137409924)